src/iOS/Xamarin.iOS/Samples/MapView/ChooseCameraController/ChooseCameraController.cs

```diff
@@ -0,0 +1,217 @@
+            GraphicsOverlay sceneGraphicsOverlay = new GraphicsOverlay()
+            {
+                SceneProperties = new LayerSceneProperties(SurfacePlacement.Absolute)
+            };
+            _mySceneView.GraphicsOverlays.Add(sceneGraphicsOverlay);
+
+            // Location at the crater.
+            MapPoint craterLocation = new MapPoint(-109.929589, 38.437304, 1700, SpatialReferences.Wgs84);
+
+            // Create the plane symbol and make it 10x larger (to be the right size relative to the scene).
+            ModelSceneSymbol planeSymbol;
+            try
+            {
+                planeSymbol = await ModelSceneSymbol.CreateAsync(_modelUri, 10.0);
+            }
+            catch (Exception ex)
+            {
+                System.Diagnostics.Debug.WriteLine(ex.Message);
+                CreateErrorDialog("Loading plane model failed. Sample failed to initialize.");
+                return;
+            }
+
+            // Create a graphic using the plane symbol.
+            Graphic planeGraphic = new Graphic(new MapPoint(craterLocation.X, craterLocation.Y, 5000.0, SpatialReferences.Wgs84), planeSymbol);
+            sceneGraphicsOverlay.Graphics.Add(planeGraphic);
+
+            // Instantiate a new camera controller which orbits a geo element.
+            _orbitPlaneCameraController = new OrbitGeoElementCameraController(planeGraphic, 300.0)
+            {
+                CameraPitchOffset = 30,
+                CameraHeadingOffset = 150
+            };
+
+            // Instantiate a new camera controller which orbits a location.
+            _orbitCraterCameraController = new OrbitLocationCameraController(craterLocation, 6000.0)
+            {
+                CameraPitchOffset = 3,
+                CameraHeadingOffset = 150
+            };
+
+            // Set the starting camera controller.
+            _mySceneView.CameraController = _orbitPlaneCameraController;
+
+            // Add the scene to the view.
+            _mySceneView.Scene = myScene;
+        }
+
+        private void ChangeCameraController(string setting)
+        {
+            // Switch on the selected text.
+            switch (setting)
+            {
+                case "Orbit plane":
+                    // Switch to the plane camera controller.
+                    _mySceneView.CameraController = _orbitPlaneCameraController;
+                    break;
+
+                case "Orbit crater":
+                    // Switch to the crater camera controller.
+                    _mySceneView.CameraController = _orbitCraterCameraController;
+                    break;
+
+                case "Free pan":
+                    // Switch to a globe camera controller, which is free pan.
+                    _mySceneView.CameraController = new GlobeCameraController();
+                    break;
+            }
+        }
+
+        public override void LoadView()
+        {
+            // Create the views.
+            View = new UIView();
```

Comment:
You'll want to explicitly set the view's background color to white to enhance the appearance of the toolbar (this will become an issue when there is nothing behind the toolbar, see comment below)